src/main/java/org/jitsi/videobridge/LipSyncHack.java

```diff
@@ -210,17 +211,37 @@ public void onRTPTranslatorWillWriteAudio(
             return;
         }
 
+        SimulcastReceiver recv = sourceVC.getTransformEngine()
+            .getSimulcastEngine().getSimulcastReceiver();
+
+        Long receiveVideoSSRC;
+        if (recv != null && recv.isSimulcastSignaled())
+        {
+            // FIXME this is a little ugly
+            receiveVideoSSRC = sourceVC.getTransformEngine()
```

Comment:
the whole logic in the spaghetti here fells like a candidate for a method in the VideoChannel, no ?